### ISSUE TITLE
I can't understand what library is used?

### ISSUE THREAD

**Issue by ghost** [changkun/applied-ml#27]
I want to use your paper : Bayesian Layers: A Module for Neural Network Uncertainty.
But I can't find what library did you use for lstm cell?
batch_size = 256
features, labels = load_dataset(batch_size)
lstm = layers.VariationalLSTMCell(512)
output_layer = tf.keras.layers.Dense(labels.shape[-1])
state = lstm.get_initial_state(features)
nll = 0.
for t in range(features.shape[1]):
net, state = lstm(features[:, t], state)
logits = output_layer(net)
nll += tf.losses.softmax_cross_entropy(
onehot_labels=labels[:, t], logits=logits)
kl = sum(lstm.losses) / batch_size
loss = nll + kl
optimizer = tf.train.AdamOptimizer()

**Comment by changkun** [comment]
Hi, sorry for the very late response. But I don't really understand what you mean. This is not a research repository but just an empty one that contains a list of papers. Thus, close.
